Dockerfile

```diff
@@ -7,12 +7,17 @@ COPY documentation/examples/prometheus.yml  /etc/prometheus/prometheus.yml
 COPY console_libraries/                     /usr/share/prometheus/console_libraries/
 COPY consoles/                              /usr/share/prometheus/consoles/
 
-RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/
+RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/ && \
+    addgroup -g 1000 prometheus && \
```

Comment:
This is only necessary if you are creating an account and want to give it a low GID.  Please see statement below for reason as to why `-S` option was not used.